Sources/StreamVideoSwiftUI/CallView/VideoRenderer.swift

```diff
@@ -111,7 +111,7 @@ public class VideoRenderer: RTCMTLVideoView {
                 return
             }
             let view = subviews.compactMap { $0 as? MTKView }.first
-            view?.preferredFramesPerSecond = 60
+            view?.preferredFramesPerSecond = ThermalStateObserver.shared.renderingFramesPerSecond
```

Comment:
Would be better if we expose the thermal state observer via our DI, not as a singleton.